src/registrar/templates/domain_detail.html

```diff
@@ -40,39 +40,50 @@
 
     {% include "includes/domain_dates.html" %}
 
+    {% if is_portfolio_user and not is_domain_manager %}
+    <div class="usa-alert usa-alert--info usa-alert--slim">
+      <div class="usa-alert__body">
+        <p class="usa-alert__text ">
+          To manage information for this domain, you must add yourself as a domain manager.
+        </p>
+      </div>
+    </div>
+    {% endif %}
+
+
     {% url 'domain-dns-nameservers' pk=domain.id as url %}
     {% if domain.nameservers|length > 0 %}
-      {% include "includes/summary_item.html" with title='DNS name servers' domains='true' value=domain.nameservers list='true' edit_link=url editable=domain.is_editable %}
+      {% include "includes/summary_item.html" with title='DNS name servers' domains='true' value=domain.nameservers list='true' edit_link=url editable=is_editable %}
     {% else %}
-      {% if domain.is_editable %}
+      {% if is_editable %}
```

Comment:
This and many other checks in the template needed to be changed from editable based solely on the state of the domain to be based on the state of the domain plus aspects of the user, so wrote a new function for that at the view level.